packages/web/config/chain-infos.ts

```diff
@@ -1806,6 +1806,33 @@ const chainInfos = (
       features: ["stargate", "ibc-transfer", "cosmwasm"],
       explorerUrlToTx: "https://scan.orai.io/txs/{txHash}",
     },
+    {
+      rpc: "http://164.90.154.41:26657",
+      rest: "http://164.90.154.41:1317",
```

Comment:
Not mandatory but you'd be better off mapping this to a DNS record so that if the IP address rotates, you don't need to file a PR again.